### ISSUE TITLE
Fox Canyon iGDE webmap (1/30)

### ISSUE THREAD

**Issue by kklausmeyer** [tnc-ca-geo/gd_intern#129]

**Comment by kklausmeyer** [comment]
Hi Connor-

I need your help to make a ArcGIS online webmap that shows the groundwater dependent ecosystems for the three groundwater basins managed by Fox Canyon.  

The data are here:  "N:\Projects\Groundwater\sites\VenturaCounty\mxds\pGDEs_FoxCanyon_MappingChanges_Connor_Oxnard.mxd"

The maps to recreate are here:  N:\Projects\Groundwater\sites\VenturaCounty\plots\FoxCanyonGDEMemo

A webmap to copy the popups is here:  Potential GDE map (v0.4):  http://arcg.is/2gffy0u

Lets talk about this before you spend too much time on it, but I think it should be pretty straight forward :)

**Comment by kklausmeyer** [comment]
Please exclude all GDEs in the Arroyo Santa Rosa groundwater basin from the webmap.  These were not in the Fox Canyon Plan so we don't want them in the webmap.  You should be able to do a definition query on the data by GW basin name.

**Comment by connorshank** [comment]
Kirk,
Here's a link to a first draft of the webmap.  Let me know how it looks, if anything needs added or changed.   For now I have shared it only internally within TNC, but that can easily be changed if its needed by external partners.
http://tnc.maps.arcgis.com/home/item.html?id=f4151608d57c4d91b955f31b0cc02b2a

**Comment by kklausmeyer** [comment]
This is great Connor-  the basic functions are all there and it makes sense.  I have the following edits:
1. Change the title to: "Indicators of Groundwater Dependent Ecosystems for the Fox Canyon Groundwater Management Agency"
2. In the contents pane, remove the text "PGDEs Foxcanyonwebmap" from all layer titles.
3. Remove the shallow alluvial aquifer boundary from the webmap- it is not needed.
4. Change "GDE Units" layer title to "Groundwater Dependent Ecosystem Grouping Area" 
5. Add a label and enable pop ups for the Groundwater Basin layer
6. Change the "GDEs" layer title to "Indicators of Groundwater Dependent Ecosystems"
7. Change the subtitle of the "GDEs" layer from "Potential GDEs" to "Local modifications"
8. In the popups for the GDE layer, remove the veg_Name field from the header of the popup.  Instead just use static text that says "Indicator of Groundwater Dependent Ecosystem"
9. Enable a pop-up for the GDE grouping layer that gives the name of the group.
10. In the GDE popups, remove the comma from the two fields that indicate the Image year so it is 2010 not 2,010.  
11. Convert the entire webmap to a webapp.  I would like it to look a little more clean and polished.  I want the legend for the GDE layer to be showing by default.  I would also like some text that describes the webmap.  Here is some draft language:  "This map shows the indicators of groundwater dependent ecosystems (iGDEs) reviewed by the Fox Canyon Groundwater Management Agency.  Each location was reviewed for accuracy and removed if it was poorly mapped.  The iGDEs were then grouped based on location and geology and then reviewed in more detail to determine the groundwater source.  For more information see http://fcgma.org/"

**Comment by kklausmeyer** [comment]
Once you make these edits, go ahead and make it public.  I would like to share with the rest of the team and that is both internal and external.

**Comment by connorshank** [comment]
I was able to make most of the requested edits, the only thing I've had trouble with is the formatting of the years.   It seems that since the attribute field is numeric, that is the only way AGO knows how to display it (with commas).    I don't think there is an easy way around this problem short of creating a new field in the original feature.   Here's a link to the web app:
http://tnc.maps.arcgis.com/apps/webappviewer/index.html?id=69639037c7f34a749cd193837a7ae5bf

**Comment by kklausmeyer** [comment]
Hi Connor-  the webmap is back on the table!  We have not heard from DWR if we can use screenshots of the tool for our webpage, so we want to have this as a back up so we can launch the website next week.  We have decided to use the Fox Canyon data, but symbolize it so that it is similar to the latest version for the iGDE database.  I have made a new MXD for you to publish here:  "N:\Projects\Groundwater\sites\VenturaCounty\mxds\pGDEs_FoxCanyonwebmap_v2_forConnor.mxd"

Basically I used defintion queries and two layers to show vegetation data only for one layer and wetland data only for another.  I also removed all the removal areas (since we did a better job of that in the latest database) .  Finally, in the MXD I'm only showing the fields that we will have in the final database.  Those are the only fields I want shown when people to the identify on the webmap.

I tried to publish the data quickly to AGO but the data wouldn't display.  It may be from the def queries.  IF so, just export to new feature classes and then publish online.

Once you publish, use the same webmap template and language that sally gave you.  Just edit any parts that are no longer relevant. I know this is a tight turn around to get this done next monday so let me know if it isn't feasible.  Tuesday would work too.
